snap/info_snap_yaml_test.go

```diff
@@ -143,9 +144,10 @@ plugs:
 	c.Check(info.Plugs, HasLen, 1)
 	c.Check(info.Slots, HasLen, 0)
 	c.Assert(info.Plugs["net"], DeepEquals, &snap.PlugInfo{
-		Snap:      info,
-		Name:      "net",
-		Interface: "network-client",
+		Snap:       info,
+		Name:       "net",
+		Interface:  "network-client",
+		SnapGlobal: true,
```

Comment:
(I understand that for testing making this attribute public is convenient).

Nitpick: Could you add a newline in this place tool. It will make the diff shorter.